GIT-VERSION-GEN

```diff
@@ -3,6 +3,9 @@
 GVF=GIT-VERSION-FILE
 DEF_VER=v2.45.0-rc0
 
+# Identify microsoft/git via a distinct version suffix
+DEF_VER=$DEF_VER.vfs.0.0
+
```

Comment:
Looks like you kept the old `DEF_VER=` to make it easier to rebase in the future? NIce.

Reply:
Yes, I was getting sick and tired of this consistent (and avoidable) merge conflict 😄

The only caveat is that care needs to be applied in case any `.0.1`, `.0.2` or `.0.3` versions should be necessary: The next rebase after such a version must reset the suffix to `.0.0`.

Reply:
Won't the trailing `.z.w` numbers key off of the tag name? We've never needed to change the `Makefile` to set those in the past.

Reply:
Oh, you're right, they _are_ derived from the tag name ;-)